src/inputs/BasicInputs/BasicRadioInput/BasicRadioInput.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Radio, RadioGroup, FormControl, FormControlLabel, TextField } from '@material-ui/core';
+import { BasicInputWrapper } from '../BasicInputWrapper';
+import PropTypes from 'prop-types';
+
+export const BasicRadioInput = (props) => {
+  const {
+    label,
+    tooltipText,
+    value,
+    textFieldProps,
+    changeRadioOption,
+    enumValues,
+    row,
+    disabled,
+    containerProps,
+    labelProps,
+    radioStyle,
+    ...otherProps
+  } = props;
+
+  return (
+    <BasicInputWrapper
+      label={label}
+      tooltipText={tooltipText}
+      containerProps={containerProps}
+      labelProps={labelProps}
+      {...otherProps}
+    >
+      {disabled ? (
+        <TextField
+          data-cy="text-field"
+          value={enumValues.find((valueOption) => valueOption.key === value).value}
+          disabled
+          {...textFieldProps}
+        />
+      ) : (
+        <FormControl>
+          <RadioGroup
+            row={row}
+            data-cy="radio-group"
+            value={value}
+            onChange={(event) => changeRadioOption(event.target.value)}
+          >
+            {enumValues.map((option) => {
+              return (
+                <FormControlLabel
+                  key={option.key}
+                  value={option.key}
+                  control={
+                    <Radio data-cy={'radio-button-' + option.value} style={radioStyle} size="small" color="primary" />
+                  }
+                  label={option.value}
+                />
+              );
+            })}
+          </RadioGroup>
+        </FormControl>
+      )}
+    </BasicInputWrapper>
+  );
+};
+
+BasicRadioInput.propTypes = {
+  /**
+   * BasicRadioInput's label
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Tooltip text
+   */
+  tooltipText: PropTypes.string,
+  /**
+   * BasicRadioInput's value
+   */
+  value: PropTypes.string.isRequired,
+  /**
+   * Additional props that you can specify for the BasicRadioInput's textField that displays the enum value selected
+   */
+  textFieldProps: PropTypes.object,
+  /**
+   * Function used when the user changes the BasicRadioInput value
+   */
+  changeRadioOption: PropTypes.func.isRequired,
+  /**
+   * List of all possible BasicRadioInput values. A value (JS object) has two attributes : **key** and **value**
+   *  `{
+         key: 'thisIsAKey',
+         value: 'This is a Value'
+       }`
+   */
+  enumValues: PropTypes.array.isRequired,
+  /**
+   * Defines if the radio group is displayed as a row or as a column.
```

Comment:
There is no default value in basic input enum. So I think we should not specify default value in basic radio input that is no more than a custom enum.

Reply:
I should be more explicit :slightly_smiling_face: I'm talking about row prop that is set to true in defaultProps, I wonder if it can be useful to specify it in documentation part :slightly_smiling_face: